bundles/org.openhab.binding.linky/src/main/resources/OH-INF/i18n/linky_fr.properties

```diff
@@ -43,4 +43,5 @@ channel-type.linky.timestamp.label = Horodatage
 
 # Thing status descriptions
 
-offline.config-error-mandatory-settings = Le nom d'utilisateur, le mot de passe et l'ID d'authentification sont obligatoires.
+offline.config-error-mandatory-settings = Le prmId est obligatoire.
```

Comment:
Don't bother of `_.frproperties` file. I will be managed within CrowdIn. The important one is `linky.properties`

Reply:
Yes you're right.
But file was already there from previous version I think.
What need to be done in this case, remove the file ?

Reply:
Yes, you can remove it.

Reply:
Ok, fix in next commit